go/extractor/dbscheme/tables.go

```diff
@@ -1150,6 +1150,20 @@ var ComponentTypesTable = NewTable("component_types",
 	EntityColumn(TypeType, "tp"),
 ).KeySet("parent", "index")
 
+// ComponentTagsTable is the table associating composite types with their component types' tags
+var ComponentTagsTable = NewTable("component_tags",
+	EntityColumn(CompositeType, "parent"),
+	IntColumn("index"),
+	StringColumn("tag"),
+).KeySet("parent", "index")
+
+// InterfacePrivateMethodIdsTable is the table associating interface types with their private method ids
```

Comment:
If I understand correctly, this table includes entries for methods (by index in the interface) which are private. If so, I don't think the comment makes that very clear. The current comment suggests that every _interface type_ has _a private method id_. How about:
```suggestion
// InterfacePrivateMethodIdsTable is the table associating interface types with the indices of their private methods.
```

Reply:
I agree it should be clearer. This is slightly more informative:
```suggestion
// InterfacePrivateMethodIdsTable is the table associating interface types with the indices and ids of their private methods.
```